### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by lucab85**
Fix typo

### CODE REVIEW

File: src/content/en/updates/2017/02/navigation-preload.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ service workers that don't have a fetch
 event](https://bugs.chromium.org/p/chromium/issues/detail?id=605844), by
 [launching service workers
 speculatively](https://codereview.chromium.org/2045153003), and other
-optimisations. However, bootup time will always be greater than zero.
+optimizations. However, bootup time will always be greater than zero.
```
Comment (mathiasbynens) [inline src/content/en/updates/2017/02/navigation-preload.md]: This is not necessarily a typo, but rather British spelling vs. American spelling. I’m personally fine with using American spelling consistently throughout the site, but I’ll defer to @jpmedley on such decisions.

File: src/content/en/updates/2017/02/navigation-preload.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ const parts = [
 </pre>
 
 Note: `Promise.resolve(event.preloadResponse)` means we get a promise for
-undefined if `event.preloadResponse` is undefined. It's a good way to normalise
+undefined if `event.preloadResponse` is undefined. It's a good way to normalize
```
Comment (mathiasbynens) [inline src/content/en/updates/2017/02/navigation-preload.md]: Same here.

File: src/content/en/updates/2017/02/navigation-preload.md [hunk 3]
```diff
@@ -335,7 +335,7 @@ the words "service worker navigation preload" in the issue Summary.
 
 <small>Many thanks to Matt Falkenhagen and Tsuyoshi Horo for their work on this
 feature, and help with this article. And a huge thanks to everyone involved in
-[the standardisation
+[the standardization
```
Comment (mathiasbynens) [inline src/content/en/updates/2017/02/navigation-preload.md]: Same here.

**Comment by mathiasbynens** [comment]
Thanks for catching these!

**Comment by petele** [comment]
@jpmedley - agreed, thoughts on this? American vs British spellings, or do we care? What does DevSite say?

**Comment by petele** [comment]
in any event - @lucab85 - thank you for your all of your contributions. Our spelling is a little, uh, well, it's not always great. So thanks for helping out.

**Comment by lucab85** [comment]
You're welcome. I really appreciate this project so I try to contribute as best as I can.

**Comment by lucab85** [comment]
IMHO this language dilemma was addressed by [Google Developer Documentation Style Guide > Language and grammar > Spelling](https://developers.google.com/style/spelling)
> In general, in cases where American spelling differs from Commonwealth/"British" spelling, **use the American spelling**.

For example: "standardi**s**e" in British English became "standardi**z**e" in American English in reference to [MW11](https://www.merriam-webster.com/dictionary/standardise)

**Comment by jakearchibald** [comment]
In future, I will try harder to comply with these standardz.

**Comment by mathiasbynens** [comment]
> IMHO this language dilemma was addressed by [Google Developer Documentation Style Guide > Language and grammar > Spelling](https://developers.google.com/style/spelling)

Ah yes, that settles it.

**Comment by lucab85** [comment]
Able for merge?

**Pull request merged**